3ds2/src/main/java/com/adyen/checkout/adyen3ds2/internal/ui/DefaultAdyen3DS2Delegate.kt

```diff
@@ -97,8 +97,11 @@ internal class DefaultAdyen3DS2Delegate(
 
     private var authorizationToken: String? by SavedStateHandleProperty(AUTHORIZATION_TOKEN_KEY)
```

Comment:
Should we get rid of the `authorizationToken` here, if we are storing `action` already?

Reply:
Let me check

Reply:
Yes just removed it